vip-mail.php

```diff
@@ -142,28 +142,34 @@ public function filter_wp_mail_from() {
 
 
 	/**
-	* This function is used to handle the wp_mail failures if the sender address is rejected by the SMTP server
-	* @param WP_Error $error The WP_Error object passed by reference
+	 * This function is used to handle the wp_mail failures if the sender address is rejected by the SMTP server
+	 *
+	 * @param WP_Error $error The WP_Error object passed by reference
 	 */
 	public function handle_wp_mail_failures( $error ) {
 		if ( defined( 'VIP_SMTP_ENABLED' ) && true === constant( 'VIP_SMTP_ENABLED' ) && isset( $error->error_data['wp_mail_failed'] ) && isset( $error->error_data['wp_mail_failed']['phpmailer_exception_code'] ) && isset( $error->errors['wp_mail_failed'] ) ) {
 			$error_data = $error->error_data['wp_mail_failed'];
 
 			// The phpmailer exception code for Sender Address rejection is 1 and we also are validating the message is matching to the one that's expected
 			if ( 1 === $error_data['phpmailer_exception_code'] && false !== strpos( $error->errors['wp_mail_failed'][0], 'Sender address rejected: not owned by user' ) ) {
-					$to          = is_array( $error_data['to'] ) ? array_map( 'esc_html', $error_data['to'] ) : esc_html( $error_data['to'] );
-					$subject     = esc_html( $error_data['subject'] );
-					$message     = esc_html( $error_data['message'] );
-					$headers     = is_array( $error_data['headers'] ) ? array_map( 'esc_html', $error_data['headers'] ) : esc_html( $error_data['headers'] );
-					$attachments = is_array( $error_data['attachments'] ) ? array_map( 'esc_url', $error_data['attachments'] ) : esc_url( $error_data['attachments'] );
+				$to          = $error_data['to'] ?? null;
+				$subject     = $error_data['subject'] ?? null;
+				$message     = $error_data['message'] ?? null;
+				$headers     = $error_data['headers'] ?? null;
+				$attachments = $error_data['attachments'] ?? null;
```

Comment:
We're removing `esc_url` from here

Reply:
Yes, I think that one should have been okay, but probably unnecessary.

Reply:
got it! thanks!